docker-compose.yaml

```
@@ -1,224 +1,223 @@
-            retries: 15
+  # Frontend
+  frontend:
+    image: "ghcr.io/camptocamp/geoshop-front:latest"
+    depends_on:
+      geoshop:
+        condition: service_healthy
+      extract:
+        condition: service_healthy
+    environment:
+      - FRONTEND_HOST=${FRONTEND_HOST:-localhost}
+      - FRONTEND_PORT=${FRONTEND_PORT:-8080}
+      - GEOSHOP_BACKEND=http://geoshop:8000
+      - EXTRACT_FRONTEND=http://extract:8080
+    volumes:
+      - geoshop-files:/geoshop/media/
+    ports:
+      - "8080:80"
+
+  # Extract
+  extract:
+    build: ./extract
+    depends_on:
+      pgsql:
+        condition: service_healthy
+      geodata:
+        condition: service_healthy
+    environment:
+      - JAVA_OPTS=-Xms1G -Xmx2G
+        -Dspring.datasource.url="jdbc:postgresql://pgsql:5432/extract"
+        -Dspring.datasource.username=extractuser
+        -Dspring.datasource.password=demopassword
+        -Duser.language=en -Duser.region=CH -Dcom.sun.jndi.ldap.connect.pool.timeout=20000
+      - GEODATA_POSTGRES_HOST=geodata
+      - GEODATA_POSTGRES_DB=geodata
+      - GEODATA_POSTGRES_USER=extractuser
+      - GEODATA_POSTGRES_PASSWORD=demopassword
+    healthcheck:
+      test: curl --fail http://localhost:8080/extract || exit 1
+      interval: 10s
+      timeout: 10s
+      retries: 15
+
+  # Extract/Postgres
+  pgsql:
+    image: postgis/postgis:14-3.2-alpine
+    environment:
+      - POSTGRES_DB=extract
+      - POSTGRES_USER=extractuser
+      - POSTGRES_PASSWORD=demopassword
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U extractuser -d postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  # Extract/Postgis
+  geodata:
+    image: postgis/postgis:14-3.2-alpine
+    environment:
+      - POSTGRES_DB=geodata
+      - POSTGRES_USER=extractuser
+      - POSTGRES_PASSWORD=demopassword
+    ports:
+      - "54320:5432"
+    volumes:
+      - ./extract/geodata:/docker-entrypoint-initdb.d/
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U extractuser -d postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  # Extract/Updatedb
+  updatedb:
+    image: postgis/postgis:14-3.2-alpine
+    depends_on:
+      extract:
+        condition: service_healthy
+    environment:
+      - PGHOST=pgsql
+      - PGDB=extract
+      - PGUSER=extractuser
+      - PGPASSWORD=demopassword
+      - GEOSHOP_BACKEND=http://geoshop:8000
+    volumes:
+      - ./updatedb:/updatedb
+    entrypoint: /updatedb/update.sh
+
+  # Geoshop
+  geoshop:
+    image: "ghcr.io/camptocamp/geoshop-api:latest"
+    depends_on:
+      postgis:
+        condition: service_healthy
+    env_file: ./geoshop/.env
+    volumes:
+      - geoshop-files:/geoshop/files
+    healthcheck:
+      test: curl --fail http://localhost:8000 || exit 1
+      interval: 10s
+      timeout: 10s
+      retries: 15
+
+  # Geoshop Front End
+  # TODO: use env variables for the backend url and port
+  geoshop_fe:
+    build: ./geoshop/front
```

This one is the correct geoshop Frontend.
The other one is something from the extract ;-) 

```suggestion
    image: "ghcr.io/camptocamp/geoshop-front:latest"
```

---

It's fixed :smile: 
I didn't pay attention of this one :see_no_evil: